app/bundles/FormBundle/Tests/FormTestAbstract.php

```diff
@@ -351,6 +354,7 @@ protected function getSubmissionModel()
             $formUploaderMock,
             $deviceTrackingService,
             new FieldValueTransformer($this->container->get('router'))
+            $dateHelper
```

Comment:
@hluchas check Travis. This is not good.

Reply:
@escopecz fixed, thank you!